### PR TITLE
feat: add pages for scisprint Taipei

### PR DESCRIPTION
**Pull request by rockleona**

### CODE REVIEW

**Comment by kuomc** [comment]
I remember that we cancel the March 23rd scisprint in Hsinchu.

**Comment by rockleona** [comment]
> I remember that we cancel the March 23rd scisprint in Hsinchu.

So I can just remove it directly?

File: content/pages/sprint/index.rst [hunk 1]
```diff
@@ -13,12 +13,18 @@ hosted by sciwork.
...
 * `(Tentative) Scisprint 2024 May in Hsinchu
   <{filename}2024/05-hsinchu.rst>`__
 
 * `(Tentative) Scisprint 2024 April in Hsinchu
   <{filename}2024/04-hsinchu.rst>`__
 
+* `(Tentative) Scisprint 2024 March in Taipei
+  <{filename}2024/03-taipei.rst>`__
+
 * `(Tentative) Scisprint 2024 March in Hsinchu
```
Comment (kuomc) [inline content/pages/sprint/index.rst:28]: I remember that we cancel the March 23rd scisprint in Hsinchu. Please help us remove this page.

**Review by kuomc — CHANGES_REQUESTED** [review]

**Comment by rockleona** [comment]
> BTW, I found a blank space between the map and contact us section. Have you encounter this situation? image

I didn't write the style properly, it's fix now, thanks for pointing out!

**Review by kuomc — APPROVED** [review]

**Pull request merged**